.github/workflows/build-reusable.yaml

```diff
@@ -221,7 +221,7 @@ jobs:
           mv riscv.tar.xz ${{ steps.toolchain-name-generator.outputs.TOOLCHAIN_NAME }}.tar.xz
       - name: strip binaries
         run: |
-          find /mnt/riscv -type f -exec strip {} \;
+          find /mnt/riscv -type f -exec sh -c 'file "$1" | grep -q "ELF" && strip "$1"' _ {} \;
```

Comment:
Please add a comment, so future readers of this line won't have a hard time understanding the purpose of this.

Reply:
Yeah, that is a good idea. I look at it now and I think, "what is this supposed to do??"

Reply:
Take a look now, let me know if that is good.